v2/api/machinelearningservices/v1api20240401/structure.txt

```diff
@@ -0,0 +1,8620 @@
+│   │   │   │   │       │   ├── ProvisioningStatus: *Enum (3 values)
+│   │   │   │   │       │   │   ├── "Completed"
+│   │   │   │   │       │   │   ├── "Failed"
+│   │   │   │   │       │   │   └── "Provisioning"
+│   │   │   │   │       │   ├── Reference: *genruntime.ResourceReference
+│   │   │   │   │       │   └── Status: *Enum (2 values)
+│   │   │   │   │       │       ├── "Disabled"
+│   │   │   │   │       │       └── "Enabled"
+│   │   │   │   │       ├── Status: *Enum (2 values)
+│   │   │   │   │       │   ├── "Disabled"
+│   │   │   │   │       │   └── "Enabled"
+│   │   │   │   │       └── TriggerType: *Enum (2 values)
+│   │   │   │   │           ├── "Cron"
+│   │   │   │   │           └── "Recurrence"
+│   │   │   │   ├── SetupScripts: *Object (1 property)
+│   │   │   │   │   └── Scripts: *Object (2 properties)
+│   │   │   │   │       ├── CreationScript: *Object (4 properties)
+│   │   │   │   │       │   ├── ScriptArguments: *string
+│   │   │   │   │       │   ├── ScriptData: *string
+│   │   │   │   │       │   ├── ScriptSource: *string
+│   │   │   │   │       │   └── Timeout: *string
+│   │   │   │   │       └── StartupScript: *Object (4 properties)
+│   │   │   │   │           ├── ScriptArguments: *string
+│   │   │   │   │           ├── ScriptData: *string
+│   │   │   │   │           ├── ScriptSource: *string
+│   │   │   │   │           └── Timeout: *string
+│   │   │   │   ├── SshSettings: *Object (2 properties)
+│   │   │   │   │   ├── AdminPublicKey: *string
+│   │   │   │   │   └── SshPublicAccess: *Enum (2 values)
+│   │   │   │   │       ├── "Disabled"
+│   │   │   │   │       └── "Enabled"
+│   │   │   │   ├── Subnet: *Object (1 property)
+│   │   │   │   │   └── Reference: *genruntime.ResourceReference
+│   │   │   │   └── VmSize: *string
+│   │   │   └── ResourceReference: *genruntime.ResourceReference
+│   │   ├── DataFactory: *Object (5 properties)
+│   │   │   ├── ComputeLocation: *string
+│   │   │   ├── ComputeType: *Enum (1 value)
+│   │   │   │   └── "DataFactory"
+│   │   │   ├── Description: *string
+│   │   │   ├── DisableLocalAuth: *bool
+│   │   │   └── ResourceReference: *genruntime.ResourceReference
+│   │   ├── DataLakeAnalytics: *Object (6 properties)
+│   │   │   ├── ComputeLocation: *string
+│   │   │   ├── ComputeType: *Enum (1 value)
+│   │   │   │   └── "DataLakeAnalytics"
+│   │   │   ├── Description: *string
+│   │   │   ├── DisableLocalAuth: *bool
+│   │   │   ├── Properties: *Object (1 property)
+│   │   │   │   └── DataLakeStoreAccountName: *string
+│   │   │   └── ResourceReference: *genruntime.ResourceReference
+│   │   ├── Databricks: *Object (6 properties)
+│   │   │   ├── ComputeLocation: *string
+│   │   │   ├── ComputeType: *Enum (1 value)
+│   │   │   │   └── "Databricks"
+│   │   │   ├── Description: *string
+│   │   │   ├── DisableLocalAuth: *bool
+│   │   │   ├── Properties: *Object (2 properties)
+│   │   │   │   ├── DatabricksAccessToken: *string
```

Comment:
This seems likely to be a secret to me

Reply:
As an aside, we should probably include "Token" in the heuristic. Can you file an item for it?

Reply:
Yep, good idea. Should also probably include "Key"?